<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>749</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>749</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>749</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bitsgap Bot Manager - Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>902</width>
      <height>731</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>902</width>
      <height>731</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>902</width>
      <height>731</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="auto_close_tab">
     <attribute name="title">
      <string>Auto Close Panel</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>0</y>
        <width>571</width>
        <height>701</height>
       </rect>
      </property>
      <property name="title">
       <string>SINGLE TRADES</string>
      </property>
      <widget class="QTableWidget" name="st_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>671</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>50</number>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>85</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Pair</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Change %</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Take Profit</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Stop Loss</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chart</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Clear</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>701</height>
       </rect>
      </property>
      <property name="title">
       <string>MULTIPLE TRADES</string>
      </property>
      <widget class="QTableWidget" name="mt_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>541</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>50</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>115</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Pair</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Change %</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>620</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Collective %:</string>
       </property>
      </widget>
      <widget class="QLabel" name="mt_collective">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>620</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mt_viewchart">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>650</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>View Chart</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>570</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Take Profit</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QLineEdit" name="mt_profit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>570</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Stop Loss</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QLineEdit" name="mt_stoploss">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>-2</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="mt_clearchart">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>650</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear Chart</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="single_system_tab">
     <attribute name="title">
      <string>Single System</string>
     </attribute>
     <widget class="QTableWidget" name="single_system_table">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>876</width>
        <height>661</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>50</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>PAIR</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>BUY</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SELL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Grid Levels</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Current Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Current Price to high price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Grid Size %</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>High Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Low Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Take Profit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stop Loss</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="single_system_inv">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLabel" name="single_system_inv_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Investment / USDT</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="singel_system_checkbox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable System</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
